docs/index.rst

```diff
@@ -205,6 +205,70 @@ Topic Management
       .. figure:: images/create_topic.png
          :alt: image
 
+7. When Confluent Monitoring Interceptors are configured on any Kafka-based clients, they write metadata to a topic called ``_confluent-monitoring``. Kafka-based clients include producers, consumers, Connect, KSQL, etc. 
```

Comment:
Is there a compelling reason that we say "Kafka-based clients" vs. just "Kafka clients"? For instance if I were referring to TIBCO EMS, I would simply say "EMS client" and likewise for NATS I would say "NATS client".

Reply:
Consider removing the word "any" -- it's unnecessary here.

Reply:
Consider changing the description of Kafka clients as follows, for clarity and precision: "Kafka clients include any application that uses the Apache Kafka client API to connect to the Kafka broker as producers or consumers, such as custom clients, Connect, KSQL, or |c3|"

Reply:
@mcqueary : spot on -- many users do not realize that Kafka Connect (for example) has embedded producers and consumers.  

I have removed `Kafka-based` and reworded very closely to your suggestion.

Thanks!